testing/simapp/app.go

```diff
@@ -170,13 +179,15 @@ type SimApp struct {
 	legacyAmino       *codec.LegacyAmino
 	appCodec          codec.Codec
 	interfaceRegistry types.InterfaceRegistry
+	MsgSvcRouter      *authmiddleware.MsgServiceRouter
```

Comment:
```suggestion
	MsgServiceRouter      *authmiddleware.MsgServiceRouter
```
I'd prefer not to shorten the name. Not shortening aligns with [sdk usage](https://github.com/cosmos/cosmos-sdk/blob/d5d8738abe548f350e81eb6087440d4d13df165b/x/auth/middleware/run_msgs.go#L16)

Reply:
You're right. I shouldn't have changed the name...